### PR TITLE
Validate yml

### PR DESCRIPTION
**Pull request by bziser**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/43453)

## Description
Added ryaml.load in yml validation

## Screenshots
image

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
please add tests and add to the PR a snippet of the error it prints.

File: demisto_sdk/commands/common/tests/structure_test.py [hunk 1]
```diff
@@ -323,6 +324,20 @@ def test_job_missing_field(self, repo, capsys, is_feed: bool, missing_field: str
         captured = capsys.readouterr().out
         assert f'Missing the field "{missing_field}" in root' in captured
 
+    def test_invalid_yml(self, capsys):
```
Comment (yuvalbenshalom) [inline demisto_sdk/commands/common/tests/structure_test.py:327]: please add standard test doc string.

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
looks good. just add standard test doc string.

**Comment by yuvalbenshalom** [comment]
@dorschw FYI

**Review by dorschw — APPROVED** [review]

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**